bin/lint

```diff
@@ -13,7 +13,10 @@ targetbin=$rootdir/target/bin
 cd "$rootdir"
 
 exe=
-if [ "$(uname -o)" = Msys ]; then
+if [ "$(uname -s)" = Darwin ]; then
+  # Darwin's uname doesn't support the -o flag so we short circuit here.
+  :;
+elif [ "$(uname -o)" = Msys ]; then
```

Comment:
```
exe=
[ "$(uname -s)" != Darwin ] && [ "$(uname -o)" = Msys ] && exe=.exe
```

Reply:
I think this is logically equivalent and I feel like what I have is a little easier to understand at a glance.